pygdf/_gdf.py

```diff
@@ -402,3 +402,48 @@ def count_nonzero_mask(mask, size):
         libgdf.gdf_count_nonzero_mask(mask_ptr, size, nnz)
 
     return nnz[0]
+
+
+def str_to_gdf_color(s):
+    """Util to convert str to gdf_color type.
+    """
+    return {
+        'green':    libgdf.GDF_GREEN,
+        'blue':     libgdf.GDF_BLUE,
+        'yellow':   libgdf.GDF_YELLOW,
+        'purple':   libgdf.GDF_PURPLE,
+        'cyan':     libgdf.GDF_CYAN,
+        'red':      libgdf.GDF_RED,
+        'white':    libgdf.GDF_WHITE,
+    }[s.lower()]
```

Comment:
Actually, it would be best if this dict was defined once globally, instead of each time the function was called.